src/app/services/Timezone.ts

```diff
@@ -0,0 +1,21 @@
+import * as lib from 'countries-and-timezones';
+import * as moment from 'moment-timezone'
+
+export class Timezone {
+
+    constructor() { }
+
+    getAllCountries() {
+        return lib.getAllCountries();
+    }
+
+    getTimezoneForCountry(id: string): lib.Timezone[] {
+        return lib.getTimezonesForCountry(id);
+    }
```

Comment:
This class smells a little bit, I understand what are you trying to do, but I don't like the class name so much because calling it like _**Timezone**_ force you to make strange things like _**lib.Timezone[]**_. I think that the main problem is the class name.

Reply:
I see that you already use it on BookingListView as TimezoneService, for now, is a good name 😛